ThreeDSecure/src/main/java/com/braintreepayments/api/threedsecure/ThreeDSecureRequest.kt

```diff
@@ -68,7 +71,8 @@ data class ThreeDSecureRequest @JvmOverloads constructor(
     var v2UiCustomization: ThreeDSecureV2UiCustomization? = null,
     var uiType: ThreeDSecureUiType = ThreeDSecureUiType.BOTH,
     var renderTypes: List<ThreeDSecureRenderType>? = null,
-    var customFields: Map<String, String>? = null
+    var customFields: Map<String, String>? = null,
+    var threeDSRequestorAppUrl: String? = null
```

Comment:
Would `requestorAppUrl` be simpler since all the params are on the `ThreeDSecureRequest` or do we prefer to keep the params aligned with Cardinal's naming? Either is fine with me

Reply:
I think leaving 3DS off of it makes sense! I will update this and the iOS PR. Agree reading it with the request name is a mouthful. 😂

Reply:
https://github.com/braintree/braintree_android/pull/1232/commits/1fcc38d4f258005a96db094b199ac37ba2286a24